uvicorn/main.py

```diff
@@ -372,12 +381,7 @@ def run(self, sockets=None, shutdown_servers=True):
     async def on_tick(self, counter) -> bool:
         # Update the default headers, once per second.
         if counter % 10 == 0:
-            current_time = time.time()
-            current_date = formatdate(current_time, usegmt=True).encode()
-            self.server_state.default_headers = [
-                (b"server", b"uvicorn"),
-                (b"date", current_date),
-            ]
```

Comment:
If we're setting up the header values in the config, then I think we end up with something pretty simple here, eg?...


```python 
current_time = time.time()
current_date = formatdate(current_time, usegmt=True).encode()
self.server_state.default_headers = [b"date", current_date)] + self.config.headers
```